leases/schema/mutations.py

```diff
@@ -78,6 +79,17 @@ def mutate_and_get_payload(cls, root, info, **input):
 
         try:
             lease = BerthLease.objects.create(**input)
+            price_group = BerthPriceGroup.objects.get_or_create_for_width(
+                berth.berth_type.width
+            )
+            product = BerthProduct.objects.get(
+                harbor=berth.pier.harbor, price_group=price_group
+            )
+            Order.objects.create(
+                customer=input["customer"], lease=lease, product=product
+            )
+        except (BerthPriceGroup.DoesNotExist, BerthProduct.DoesNotExist) as e:
```

Comment:
hm, since you do `BerthPriceGroup.objects.get_or_create_for_width` you will not encounter `BerthPriceGroup.DoesNotExist` 🤔  so, either switch to `BerthPriceGroup.objects.get` or skip this exception

Reply:
Right!! I forgot to remove it from my testings. 😅